platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -20,55 +72,55 @@
 
 @property (nonatomic) BOOL iconOptional;
```

Comment:
I think we should give it a try at least, if we decide not to go with `NSNumber`